pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -108,6 +108,15 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 		logger.Error(err, "unable to get MasterUserRecord")
 		return reconcile.Result{}, err
 	}
+
+	// TODO remove this code block once all MUR resources have had their labels updated
+	mur, err = r.migrateLabels(mur)
```

Comment:
Shouldn't we return here if the mur was migrated/updated? So the next reconcile is triggered when the actual update happens?

Reply:
Sure, that makes sense.  I'll have it return a boolean also.  As for the `UserSignup.Name` -> `Spec.UserID` migration, I *think* we could possibly do this in a separate PR...  actually though, looking at the code it's not really clear where we will use the `UserSignup.Spec.UserID` value.  We're now setting the `owner` label on MUR to point to the UserSignup name, but I'm not certain about the purpose for UserID.